### ISSUE TITLE
Is it possible to limit the zoom in so that there is never greater than a 1 to 1 pixel ratio from the image to the stencil sizes? 

### ISSUE THREAD

**Issue by monkeytroy** [Norserium/vue-advanced-cropper#101]
On a stencil restricted crop where I have specified a stencil w and h...  I would like to not allow the crop area to be more than a 1 to 1 pixel size in the stencil.   

So for a 100x100 stencil and a 200x200 image, I want to be able to zoom out so that the full 200x200 pixels were fit into the 100x100 crop area.   But when I zoom out I never want the part of the image in the stencil to be less than 100x100.   The most you could zoom out to the full 200x200 size where what is in the box is truly 100x100 and not less pixel stretched to 100x100.  

Sorry, not sure a better way to ask this.   Tried various things but can't seem to figure out a way to make this work.   

Any advice?

**Comment by monkeytroy** [comment]
Of course as soon as I send this I figure it out...   but why does this behave differently than setting min-height and min-width on the component?

:size-restrictions-algorithm="percentsRestriction"

`
percentsRestriction({ minWidth, minHeight, maxWidth, maxHeight, imageWidth, imageHeight }) {
      return {
          minWidth: 100,
          minHeight: 100,
          maxWidth: maxWidth,
          maxHeight: maxHeight,
      };
  },
`

**Comment by Norserium** [comment]
@monkeytroy, hello! What's the library version do you use?

**Comment by Norserium** [comment]
@monkeytroy, any news?

**Comment by monkeytroy** [comment]
Yup.. that was it.  minWidth and minHeight work fine on 1.0.0.   Ty much sir.

**Comment by Norserium** [comment]
@monkeytroy, just for your information, the default `sizeRestrictionsAlgorithm` limits the size in percents relative to image size. So `minWidth`, `maxWidth`, `minHeight` or `maxHeight` just sets the restriction relative to image (for example, 25%, 50% and etc of the image).

This behavior was changed, and now the restrictions `minWidth`, `maxWidth`, `minHeight` or `maxHeight` sets the restriction in pixels, just like you want (and most of users also).
